src/main/java/nl/hsac/fitnesse/fixture/slim/EmailFixture.java

```diff
@@ -0,0 +1,177 @@
+package nl.hsac.fitnesse.fixture.slim;
+
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.mail.*;
+import javax.mail.internet.MimeMultipart;
+import javax.mail.search.SearchTerm;
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class EmailFixture extends SlimFixture {
+    private final static Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+    private static final Date HOURS_BACK = new Date(System.currentTimeMillis() - 1000 * 60 * 60 * 24);
+    private Store store;
+
+    public EmailFixture() {
+
+    }
+
+    /**
+     * | set mail provider with host | <i>host</i> | port | <i>port</i> | user | <i>username</i> | password | <i>password</i> |
+     */
+    public void setMailProviderWithHostPortUserPassword(final String host, final String port, final String username, final String password) {
+        final Properties props = new Properties();
+        props.setProperty("mail.smtp.host", host);
+        props.setProperty("mail.smtp.socketFactory.port", port);
+        props.setProperty("mail.smtp.socketFactory.class", "javax.net.ssl.SSLSocketFactory");
+        props.setProperty("mail.smtp.auth", "true");
+        props.setProperty("mail.smtp.port", port);
```

Comment:
smtp is for sending mail. Should this not be pop/imap(s) instead?

Reply:
correct!